dotnet-desktop-guide/framework/winforms/index.yml

```diff
@@ -1,11 +1,11 @@
 ### YamlMime:Landing
 
 title: .NET Desktop Guide for Windows Forms
-summary: Learn about using Windows Forms on Windows with either .NET Framework, .NET 5 and above, or .NET Core 3.1.
+summary: Learn about using Windows Forms (WinForms), a graphical user interface for Windows and .NET Framework.
```

Comment:
Not sure how I feel about "Learn about using", consider either "Learn how to use ..." or "Learn about ..."